lib/src/modules/bus.dart

```diff
@@ -23,9 +23,11 @@ class BusSubset extends Module with InlineSystemVerilog {
   late final String _subsetName;
 
   /// The input to get a subset of.
-  late final Logic _original;
+  ///
+  /// This should not be used outside of this module.
+  late final Logic original;
```

Comment:
Then why is it not private if it should not be used outside?

Reply:
I updated the message here, but the idea is that it's useful to know the input and/or input name for these modules, even though you shouldn't connect to them.